bitbucket-scm-filter-aged-refs/src/test/java/org/jenkinsci/plugins/scm_filter/BitbucketAgedRefsTraitTest.java

```diff
@@ -1,30 +1,58 @@
 package org.jenkinsci.plugins.scm_filter;
 
-import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
Please don't switch the assertion framework.

Reply:
Resolved.